vendor/modules.txt

```diff
@@ -196,7 +196,6 @@ github.com/operator-framework/operator-sdk/pkg/kube-metrics
 github.com/operator-framework/operator-sdk/pkg/leader
 github.com/operator-framework/operator-sdk/pkg/log/zap
 github.com/operator-framework/operator-sdk/pkg/metrics
-github.com/operator-framework/operator-sdk/pkg/restmapper
```

Comment:
Let's figure out why the imports moved. I want to avoid moving them back and forth with PRs, so if there is `go fmt` or similar is sorting them, let's go with that. Otherwise, avoid rearranging.

Reply:
It appeared when I used sdk 0.14.1

Reply:
I'm talking about lines going up and down... not imports being added/removed.

Reply:
I should have made this comment further down... for example, look at lines 406,-407 versus 409 and 415

Reply:
Line movements seems to be generated this way with new go versions.